ERROR 2022-09-28 18:53:21,461 connection.py _receive_response 206 [7fd04840-9e3f-4293-a141-1406418f088a]: no such element
ERROR 2022-09-28 18:53:22,993 assertbase.py _teardown_collect 273 <bound method FirstTest.test_11_all_unbing_device of <minitest.FirstTest testMethod=test_11_all_unbing_device>>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/minium/framework/libs/unittest/case.py", line 32, in testPartExecutor
    yield
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/minium/framework/libs/unittest/case.py", line 118, in run
    self._callTestMethod(testMethod)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 549, in _callTestMethod
    method()
  File "/Users/isentech/Desktop/python/miniu_test/minitest.py", line 93, in test_11_all_unbing_device
    self.page.get_element('/page/view/view[2]/button').click()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/minium/miniprogram/base_driver/minium_log.py", line 135, in wrapper
    result = func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/minium/miniprogram/base_driver/page.py", line 200, in get_element
    return self.get_element_by_xpath(
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/minium/miniprogram/base_driver/minium_log.py", line 135, in wrapper
    result = func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/minium/miniprogram/base_driver/page.py", line 387, in get_element_by_xpath
    raise MiniElementNotFoundError("element[%s] not found" % xpath)
minium.framework.exception.MiniElementNotFoundError: element[/page/view/view[2]/button] not found
